core/blocksigner/enclave.go

```diff
@@ -42,6 +43,12 @@ func (ec EnclaveClient) Sign(ctx context.Context, pk ed25519.PublicKey, bh *lega
 		client.BaseURL = tup[0]
 		client.AccessToken = tup[1]
 
+		// If the provided access token is just a password with no
+		// user, use a default basic auth username of 'chaincore'.
+		if !strings.Contains(client.AccessToken, ":") {
+			client.AccessToken = "chaincore:" + client.AccessToken
+		}
```

Comment:
Hrm I wish we could just require it to have a colon and/or use it exactly as presented. Do we have folks in the wild who are expecting a particular behavior?